source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -117,6 +128,27 @@ const doSort = (answers: Answer[], sortField: string): Answer[] =>
   [...answers].sort(
     (a: Answer, b: Answer) => Number(a[sortField]) - Number(b[sortField])
   );
+
+/**
+ * Returns the index of the last element in the array where predicate is true, and -1
+ * otherwise.
+ * @param array The source array to search in
+ * @param predicate find calls predicate once for each element of the array, in descending
+ * order, until it finds one where predicate returns true. If such an element is found,
+ * findLastIndex immediately returns that element index. Otherwise, findLastIndex returns -1.
+ */
+function findLastIndex<T>(
+  array: Array<T>,
+  predicate: (value: T, index: number, obj: T[]) => boolean
+): number {
+  let l = array.length;
+  // eslint-disable-next-line no-plusplus
+  while (l--) {
+    if (predicate(array[l], l, array)) return l;
+  }
+  return -1;
+}
```

Comment:
I guess you can remove the //eslint-disable ... by writing like this 

```js
  let l = array.length;
  while (l) {
    if (predicate(array[l], l, array)) return l;

     l -=1
  }
  return -1;
}
```

Reply:
While it does work, I'm unsure how it affects readability of the code. Why are we even using the [no-plusplus](https://eslint.org/docs/rules/no-plusplus) rule in the first place?

Reply:
Disabled rule